dali/base/dacoven.cpp

```diff
@@ -936,6 +936,7 @@ ICoven &queryCoven()
 {
     if (coven==NULL)
     {
+        PrintStackReport();
```

Comment:
not directly related, but added to debug something I hit when implementing this PR, and think it is worth leaving in.

Reply:
No danger of crating excessive tracing? Probably not if this sutuation is always fatal...

Reply:
Should always be fatal.
It would still be useful if in _DEBUG only..
leave it or make it _DEBUG only?